### DISCUSSION TITLE
A test fatal for help "fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree."

### DISCUSSION THREAD

**Discussion by 1749335863** [ImperialCollegeLondon/sharpy#189] [General]
Dear scholars,

When I followed the SHARPy Installation Guide and started to test sharpy, I found it tolde me that
 "fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'"
It did show the logal of  "SHARPY", which made me believe I had sucessfully installed SHARPy.
![1](https://user-images.githubusercontent.com/65592096/165015147-ae2a7ce9-ea98-4366-ae2a-749c778159c5.png)
However,when I tried tp run the "cantilever.sharpy" generated by "model_static_cantilever.py", which was given in sharpy, it occoured the same fatal as follows.
![2](https://user-images.githubusercontent.com/65592096/165015440-302fe4d2-a31a-4947-b281-4914732b1691.png)
Could you help me?
Ubuntu 18.04 on VM
 

Yours,
Wen

**Comment by ngoiz** [comment]
Hi Wen,

This is an odd one that has to do with SHARPy trying to show the git revision it is running. The only thing I can think of is that your SHARPy is actually a git repository (i.e. you can run `git status` from the directory without an error). Try that see if it runs ok, if not, the only thing I can suggest is to reinstall a clean copy.

Thanks,
Norberto
